mmocr/datasets/pipelines/ner_transforms.py

```diff
@@ -0,0 +1,60 @@
+import torch
+
+from mmdet.datasets.builder import PIPELINES
+from mmocr.models.builder import build_convertor
+
+
+@PIPELINES.register_module()
+class NerTransform:
+    """Ner transform ."""
+
+    def __init__(self, label_convertor=None, max_len=None):
+        assert label_convertor is not None
+        self.label_convertor = build_convertor(label_convertor)
+        self.max_len = max_len
```

Comment:
From L22, we know that `max_len` cannot be None right?

Either make an assertion that `max_len` is valid here, or make `max_len` a required positional argument without `=None`